docs/glossary.rst

```diff
@@ -0,0 +1,400 @@
+:ref:`subscription plan<glossary-subscription-plan>` and scaling, each cluster
+will have a certain storage capacity and can process a certain amount of
+ingests and queries per second. Only actual cluster usage is billed.
+
+A cluster has a name, a unique ID, as well as a storage and processing
+capacity and a number of nodes. Note that clusters are also versioned. For
+information on how to deploy a cluster, please see our `tutorial for deploying
+a CrateDB cluster via Azure Marketplace`_.
+
+
+.. _glossary-console:
+
+Console
+-------
+
+The CrateDB Cloud Console is the hosted user interface for CrateDB Cloud. It
+is a fully supported, easy-to-use UI which allows customers to interact with
+every aspect of the CrateDB Cloud service (subject to :ref:`user role
+permissions<user-roles>`.) While CrateDB Cloud also supports a :ref:`CLI
+<glossary-cli>` for interacting with the service, we assume use of the Console
+by default. Only the Console allows deployment of a :ref:`cluster
+<glossary-cluster>`.
+
+For information on how to use specific elements of the Console, refer to our
+`Howtos`_.
+
+
+.. _glossary-consumer:
+
+Consumer
+--------
+
+A consumer in the sense used for CrateDB Cloud architecture and documentation
+is an entity that reads event data from an event hub. It is possible to use a
+consumer, such as Azure EventHub, with CrateDB Cloud: you can store the data
+processed by the consumer on the Cloud :ref:`cluster<glossary-cluster>`. For a
+tutorial on how to do this, see `this article on our blog`_. Operations on
+consumers are registered in the :ref:`Audit Log<glossary-audit-log>`.
+
+
+.. _glossary-croud:
+
+Croud
+-----
+
+Croud is the name of the CrateDB Cloud Command Line Interface (:ref:`CLI
+<glossary-cli>`). You can use Croud to interact with the :ref:`organization
+<glossary-org>`, :ref:`projects<glossary-project>` and :ref:`products
+<glossary-product>` you have access to. Croud is intended for customers who
+prefer a CLI to the use of a hosted web interface such as the CrateDB Cloud
+:ref:`Console<glossary-console>`. Note however that the Console is the default
+way to interact with CrateDB Cloud, and currently clusters can only be deployed
+within the Console. The Documentation for Croud can be found under the `Croud
+CLI header`_ in the Docs sidebar.
+
+
+.. _glossary-DTU:
+
+DTU
+---
+
+DTU stands for Database Transaction Unit. CrateDB Cloud uses DTUs to create
+configurations (combinations) of hardware specifications for specific
+:ref:`subscription plans<glossary-subscription-plan>`. The advantage is that
+the customer does not need to specify every element of the hardware
+configuration themselves, but can simply identify the price per DTU for a given
+plan and see how it matches their use case. Also, each additional :ref:`scale
+unit<glossary-scale-unit>` for a :ref:`cluster<glossary-cluster>` in a given
+plan corresponds to one (1) additional DTU. This makes using the CrateDB Cloud
```

Comment:
You are somehow right, and aren't :) That was the case, but changed for the dev offer. SU 1: 3 DTUs, SU2: 4 DTUs , SU3: 5 DTUs. That's why each Scale unit doesn't correspond to 1 additional DTU anymore

Reply:
But 3 to 4 to 5 is still one _additional_ DTU per scale unit? It's certainly true the dev one doesn't start at 1, but that's a different thing. Though I could add that